http_requests.py

```diff
@@ -212,3 +220,30 @@ def delete(self, url: str, return_type=None, **kwargs):
                 return_type=None,
                 **kwargs
         )
+
+
+def _log_stacktrace_to_els(url: str):
+    if not ci.util._running_on_ci():
+        return  # early exit if not running in ci job
+
+    els_index = 'requests_stacktrace'
+    config_set = 'internal_active'
+    try:
+        config_set = ci.util.ctx().cfg_factory().cfg_set(config_set)
+        elastic_cfg = config_set.elasticsearch()
+
+        now = datetime.datetime.utcnow()
+        json_body = {
+            'date': now.isoformat(),
+            'url': url,
+            'stacktrace': traceback.format_stack()
```

Comment:
if possible, do some parsing for common error classification (e.g. discriminate between "TLS handshake-err" and "connection closed" err, and add a hint to this to a separate field (e.g. error_class = tls_handshake|connection_closed|other, ...)

Reply:
also, a separate hostname attr could be worthwhile (urllib.parse.urlparse....) -> would allow us to state statistics about affected target hosts.

also, add from-host (communication is done between two endpoints, usually..)